src/classes/User/PassageUser.ts

```diff
@@ -0,0 +1,290 @@
+     */
+    public async activate(userId: string): Promise<PassageUser> {
+        try {
+            const response = await this.usersApi.activateUser({
+                userId,
+                appId: this.config.appId,
+            });
+            return response.user;
+        } catch (err) {
+            if (err instanceof ResponseError) {
+                throw await PassageError.fromResponseError(err, 'Could not activate user');
+            }
+            throw err;
+        }
+    }
+
+    /**
+     * Deactivate a user using their user ID.
+     *
+     * @param {string} userId The Passage user ID
+     * @return {Promise<PassageUser>} Passage User object
+     */
+    public async deactivate(userId: string): Promise<PassageUser> {
+        try {
+            const response = await this.usersApi.deactivateUser({
+                userId,
+                appId: this.config.appId,
+            });
+
+            return response.user;
+        } catch (err) {
+            if (err instanceof ResponseError) {
+                throw await PassageError.fromResponseError(err, 'Could not deactivate user');
+            }
+
+            throw err;
+        }
+    }
+
+    /**
+     * Update a user.
+     *
+     * @param {string} userId The passage user ID
+     * @param {UpdateUserRequest} options The user attributes to be updated
+     * @return {Promise<PassageUser>} Pasasge User Object
+     */
+    public async update(userId: string, options: UpdateUserRequest): Promise<PassageUser> {
+        try {
+            const response = await this.usersApi.updateUser({
+                userId,
+                appId: this.config.appId,
+                updateUserRequest: options,
+            });
+
+            return response.user;
+        } catch (err) {
+            if (err instanceof ResponseError) {
+                throw await PassageError.fromResponseError(err, 'Could not update user');
+            }
+
+            throw err;
+        }
+    }
+
+    /**
+     * Create a user.
+     *
+     * @param {CreateUserRequest} args Arguments for creating a user
+     * @return {Promise<PassageUser>} Passage User object
+     */
+    public async create(args: CreateUserRequest): Promise<PassageUser> {
+        try {
+            const response = await this.usersApi.createUser({
+                appId: this.config.appId,
+                createUserRequest: args,
+            });
+
+            return response.user;
+        } catch (err) {
+            if (err instanceof ResponseError) {
+                throw await PassageError.fromResponseError(err, 'Could not create user');
+            }
+
+            throw err;
+        }
+    }
+
+    /**
+     * Delete a user using their user ID.
+     *
+     * @param {string} userId The userID used to delete the corresponding user.
+     * Either an E164 phone number or email address.
+     * @return {boolean} True if user was deleted, false if not
```

Comment:
```suggestion
     * @return {Promise<boolean>}
```

we will never return false since a failed deletion will just throw. but can probably just keep this blank to match the other boolean return docstrings